ArduCopter/mode_auto.cpp

```diff
@@ -277,7 +277,7 @@ void ModeAuto::takeoff_start(const Location& dest_loc)
     set_throttle_takeoff();
 
     // get initial alt for WP_NAVALT_MIN
-    auto_takeoff_set_start_alt();
+    auto_takeoff_set_start_and_final_alt(alt_target);
```

Comment:
This is discarding the altitude frame. I think it should be:

```suggestion
   IGNORE_RETURN(dest.get_alt_cm(Location::AltFrame::ABOVE_ORIGIN, alt_target));
   auto_takeoff_set_start_and_final_alt(alt_target);
```
I think navigation is always above origin?

Reply:
Randy, I will need your help here.

Reply:
@IamPete1, yes, the position controller and wpnav always use alt above EKF origin if they're consuming a Vector3f.

Reply:
Has this been addressed?